bindata/scripts/kargs.sh

```diff
@@ -9,8 +9,67 @@ args=$(chroot /host/ cat /proc/cmdline)
 
 IS_OS_UBUNTU=true; [[ "$(chroot /host/ grep -i ubuntu /etc/os-release -c)" == "0" ]] && IS_OS_UBUNTU=false
```

Comment:
question how this one will work when we run on systemd mode?
the /host folder doesn't exist in the host itself only inside the container.